### PR TITLE
Add beautifulsoup

### PR DESCRIPTION
**Pull request by Onager**

### CODE REVIEW

File: dependencies.ini [hunk 1]
```diff
@@ -4,6 +4,13 @@ minimum_version: 20190305
 rpm_name: python2-artifacts
 version_property: __version__
 
+[beautifulsoup4]
```
Comment (joachimmetz) [inline dependencies.ini]: Have a loot at https://github.com/google/timesketch/pull/790/files#diff-32f6ea337b09b9bc597215dddf395d89R46

the name of the module is bs4 not beautifulsoup4

Comment (Onager) [reply dependencies.ini]: OK, the name doesn't match in l2tdevtools then.

Comment (joachimmetz) [reply dependencies.ini]: add `l2tbinaries_name: beautifulsoup4`

**Comment by joachimmetz** [comment]
Per conversation, finding an alternative solution instead of depending on beautifulsoup

**Pull request closed without merging**